README.md

```diff
@@ -30,9 +30,61 @@ In Node.js
     console.log("Public Rooms: %s", JSON.stringify(data));
   });
 ```
+
 See below for how to include libolm to enable end-to-end-encryption. Please check
 [the Node.js terminal app](examples/node) for a more complex example.
 
+To start the client:
+
+```javascript
+client.startClient({initialSyncLimit: 10});
```

Comment:
With lazy-loading, this now returns a promise, so one would either need to `await client.startClient(...);` or use `then` like so: `client.startClient(...).then(() => {/* work with client here*/});`. First option is probably preferable.